app/views/matches/show.html.erb

```diff
@@ -0,0 +1,14 @@
+<h1>Match</h1>
+<p>
+    <% if Time.zone.now.after?(@match.kickoff_at) %>
+        <%= @match.home_team.name %> <%= @match.display_score %> <%= @match.away_team.name %>
+    <% else %>
+        <%= @match.home_team.name %> v. <%= @match.away_team.name %>
+    <% end %>
+</p>
+<p>
+    <%= I18n.l(@match.kickoff_at, format: :with_day_of_week) %>
+</p>
+<p>
+    <%= @match.stadium %>
```

Comment:
Stadium should use locale setting (English name and long name needs to be added to current data)